.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -157,6 +157,10 @@ parameters:
     displayName: Build 2404 Gen2 Containerd
     type: boolean
     default: true
+  - name: build2404tlgen2containerd
+    displayName: Build 2404 TL Gen2 Containerd
+    type: boolean
+    default: false
```

Comment:
false for now because its not in use yet right?

Reply:
that is correct, yes.